root/usr/share/nethesis/NethServer/Module/Mail/Domain/Modify.php

```diff
@@ -64,6 +71,21 @@ public function validate(\Nethgui\Controller\ValidationReportInterface $report)
         }
     }
 
+    public function readDkimFile()
+    {
+        if ( ! isset($this->parameters['domain'])) {
+            return '';
+        }
+        $fileName =  self::DKIM_PATH . $this->parameters['domain'] . '.txt';
+        $value = $this->getPhpWrapper()->file_get_contents($fileName, FALSE, NULL, -1, self::DKIM_MAX_LENGTH);
```

Comment:
Do not limit the read size: let's read the whole file contents!

Reply:
done